### PR TITLE
Fixed schema objects ordering

### PR DESCRIPTION
**Pull request by sebjulliand**
Fixes https://github.com/codefori/vscode-db2i/issues/197

The query run when fetching the objects for the schemas browser was supposed to order the items alphabetically but the `Select * (...) Union All` applied to join the queries would prevent it.

This PR fixes this and restores the alphabetical order in the schemas browser.

### CODE REVIEW

**Review by chrjorgensen — APPROVED** [review]
@sebjulliand Good catch - Mr. SqlGuru! 😉 

I approved - but it seems not to be accepted by GitHub... could it be me not having write access to this repo?

File: src/database/schemas.ts [hunk 1]
```diff
@@ -35,10 +33,9 @@ export default class Schemas {
       switch (type) {
         case `schemas`:
           selects.push([
-            `select '${type}' as OBJ_TYPE, SCHEMA_NAME as NAME, SCHEMA_TEXT as TEXT, SYSTEM_SCHEMA_NAME as SYS_NAME, '' as SYS_SCHEMA, '' as SPECNAME, '' as BASE_SCHEMA, '' as BASE_OBJ`,
+            `select '${type}' as OBJ_TYPE, QSYS2.DELIMIT_NAME(SCHEMA_NAME) as NAME, SCHEMA_TEXT as TEXT, SYSTEM_SCHEMA_NAME as SYS_NAME, '' as SYS_SCHEMA, '' as SPECNAME, '' as BASE_SCHEMA, '' as BASE_OBJ`,
```
Comment (davecharron) [inline src/database/schemas.ts:36]: Delimiting the schema name here breaks the SchemaBrowser `vscode-db2i.manageSchemaBrowserList` action, which itself applies delimiting to the schema names.  That then trickles through the schema browser tree as the name ends up double-delimited.  The SchemaBrowser handles the sorting of the schemas in its `getChildren` method.

**Review by davecharron — CHANGES_REQUESTED** [review]

File: src/database/schemas.ts [hunk 2]
```diff
@@ -131,15 +119,14 @@ export default class Schemas {
...
           `from QSYS2.SYSTYPES`,
           `where USER_DEFINED_TYPE_SCHEMA = '${schema}' ${details.filter ? `and UPPER(USER_DEFINED_TYPE_NAME) like '%${details.filter}%'`: ``}`,
-          `order by USER_DEFINED_TYPE_NAME asc`
         ].join(` `));
         break;
       }
     }
 
-    query = selects.map(s => `select * from (${s})`).join(`\n UNION ALL \n`);
+    const query = `${selects.join(" UNION ALL ")} Order by NAME asc`;
```
Comment (davecharron) [inline src/database/schemas.ts:127]:
```suggestion
    const query = `${selects.join(" UNION ALL ")} Order by QSYS2.DELIMIT_NAME(NAME) asc`;
```

**Comment by sebjulliand** [comment]
Thank you for your review and your suggestion @davecharron 🙏🏻
You can give it another go if you wish.

**Review by davecharron — APPROVED** [review]
Looks good to go!

**Comment by worksofliam** [comment]
Tests are passing. Thanks @sebjulliand!!

**Pull request merged**